mpf/crawler.py

```diff
@@ -0,0 +1,50 @@
+import requests
+from pyexcel_ods import get_data 
+from pyexcel_xls import get_data 
+import pathlib 
+
+
```

Comment:
Lembrar de começar o tratamento de erros, é uma fase importante do desenvolvimento. Mas podemos começar resolvendo os outros comentários desse pr, depois a gente começa outro com o tratamento dos erros. =D

Reply:
Tem algum padrão especifico ? Ou apenas levantar exceções nos locais propícios a error ?

Reply:
#122

Reply:
[Dealing with errors](https://www.programiz.com/python-programming/exception-handling#:~:text=In%20Python%2C%20exceptions%20can%20be,we%20have%20caught%20the%20exception.)
Dá uma olhadinha nesse link, thyago, acredito que a forma de try, raise e except é o melhor jeito de tratar erros em python. =D